packages/autocomplete/package.json

```diff
@@ -45,15 +45,16 @@
     "jest-standard-reporter": "^1.1.1",
     "jest-styled-components": "^6.3.4",
     "npm-run-all": "^4.1.5",
-    "pcln-design-system": "^3.7.1",
-    "pcln-icons": "^3.2.0",
+    "pcln-design-system": "^4.0.0",
+    "pcln-icons": "^4.0.0",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
     "rimraf": "^3.0.2",
-    "styled-components": "^3.4.10",
+    "styled-components": "^4.4.1",
     "us": "^2.0.0"
   },
   "peerDependencies": {
-    "pcln-design-system": "^3.7.1"
+    "pcln-design-system": "^4.0.0",
```

Comment:
Should this require `react` as a peer too?

Reply:
I think we can leave it out here because they'll get the same warning if they're missing the same peer that's required by `pcln-design-system` core.